backend/api/generator.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+# Copyright 2018-2020 Google LLC
```

Comment:
How do we generally do with updating licenses?

Reply:
We usually update the date to include the last year there's a change, so we should update this as
`Copyright 2021 The Kubeflow Authors`, because you created it.